### PR TITLE
Ins 29592

### PR DESCRIPTION
**Pull request by RefaelAdi**
Plug-ins readme updates: for every plug-in that supported in GI/SaaS - added a section that refers to the guide on how to configure UC connection in GI

### CODE REVIEW

File: filter-plugin/logstash-filter-azure-apachesolr-guardium/README.md [hunk 1]
```diff
@@ -195,3 +195,7 @@ The Guardium universal connector is the Guardium entry point for native audit lo
   - LOGIN_FAILED    : Not available with logs
 
 - While launching Solr in SolrCloud mode, multiple logs will be generated for single query execution as a call to shard(In SolrCloud, a logical partition of a single Collection) and replica(A core that acts as a physical copy of a shard in a SolrCloud Collection). 
+
+## 5. Configuring the Apache Solr filters in Guardium Insights
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
+In the ```Input configuration``` section, refer to the Filebeat section.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-azure-apachesolr-guardium/README.md:201]:
```suggestion
For the input configuration step, refer to the [Filebeat section](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md#Filebeat-input-plug-in-configuration).
```

File: filter-plugin/logstash-filter-aurora-mysql-guardium/README.md [hunk 2]
```diff
@@ -140,3 +140,9 @@ The Guardium universal connector is the Guardium entry point for native audit lo
 8. The "type" fields should match in the input and the filter configuration sections. This field should be unique for every individual connector added.
 9. Click Save. Guardium validates the new connector, and enables the universal connector if it was
 	disabled. After it is validated, it appears in the Configure Universal Connector page.
+
+## Configuring the Aurora-MySQL Guardium Logstash filters in Guardium Insights
+
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
+
+In the input configuration section, refer to the CloudWatch_logs section.
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-aurora-mysql-guardium/README.md:148]: Why isn't it a link like the rest of the plugins

File: filter-plugin/logstash-filter-cassandra-guardium/README.md [hunk 3]
```diff
@@ -130,3 +130,7 @@ The Guardium universal connector is the Guardium entry point for native audit lo
 7. Update the filter section to add the details from the [filter-test-beats.conf](https://github.com/IBM/universal-connectors/blob/main/filter-plugin/logstash-filter-cassandra-guardium/filter-test-beats.conf) file's filter part, omitting the keyword "filter{" at the beginning and its corresponding "}" at the end.
 8. The "type" fields should match in the input and the filter configuration sections. This field should be unique for  every individual connector added.
 9. Click Save. Guardium validates the new connector, and enables the universal connector if it was disabled. After it is validated, it appears in the Configure Universal Connector page.
+
+## 5. Configuring the Cassandra filters in Guardium Insights
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
+In the ```Input configuration``` section, refer to the Filebeat section.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-cassandra-guardium/README.md:136]:
```suggestion
For the input configuration step, refer to the [Filebeat section](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md#Filebeat-input-plug-in-configuration).
```

File: filter-plugin/logstash-filter-documentdb-aws-guardium/README.md [hunk 4]
```diff
@@ -123,3 +123,9 @@ disabled. After it is validated, it appears in the Configure Universal Connector
      - Client HostName : Not available with Audit/Profier logs
 - Server IPs are also not reported because they are not part of the audit stream. That said, the "add_field" clause in the configuration adds a user defined Server Host Name that can be used in reports and policies if desired.
 - Because Sniffer saves the DB name once when a new session is created, and not with every event, DB name will be updated and populated correctly in Guardium only when everytime a new database connection is established with database name. If Database connection is established without database name, then the database on which the first query for that session runs, will be retained in Guardium. Even if user switches between the databases for the same session.     
+
+## Configuring the DocumentDB Guardium Logstash filters in Guardium Insights
+
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
+
+In the input configuration section, refer to the CloudWatch_logs section.
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-documentdb-aws-guardium/README.md:131]: same comment

File: filter-plugin/logstash-filter-couchdb-guardium/README.md [hunk 5]
```diff
@@ -122,3 +122,6 @@ disabled. After it is validated, it appears in the Configure Universal Connector
     - Client port : Not available with logs
     - Client HostName : Not available with logs  
 
+## 5. Configuring the Couchdb filters in Guardium Insights
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
+In the ```Input configuration``` section, refer to the Filebeat section
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-couchdb-guardium/README.md:127]:
```suggestion
For the input configuration step, refer to the [Filebeat section](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md#Filebeat-input-plug-in-configuration).
```

File: filter-plugin/logstash-filter-postgres-guardium/AwsPostgres_README.md [hunk 6]
```diff
@@ -202,3 +202,9 @@ The Guardium universal connector is the Guardium entry point for native audit lo
 8. The "type" fields should match in the input and the filter configuration sections. This field should be unique for  every individual connector added.
 9. Click Save. Guardium validates the new connector, and enables the universal connector if it was
 	disabled. After it is validated, it appears in the Configure Universal Connector page.
+
+## Configuring the Postgres AWS Guardium Logstash filters in Guardium Insights
+
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
+
+In the input configuration section, refer to the CloudWatch_logs section.
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-postgres-guardium/AwsPostgres_README.md:210]: same comment

File: filter-plugin/logstash-filter-mariadb-guardium/README.md [hunk 7]
```diff
@@ -192,3 +192,7 @@ disabled. After it is validated, it appears in the Configure Universal Connector
     - OS User  - Not available in Audit logs
     - ClientIP - Not avaiable in Audit Logs
     - Source Program - Not available in Audit Logs
+
+## 5. Configuring the Mariadb filters in Guardium Insights
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
+In the ```Input configuration``` section, refer to the Filebeat section
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-mariadb-guardium/README.md:198]:
```suggestion
For the input configuration step, refer to the [Filebeat section](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md#Filebeat-input-plug-in-configuration).
```

**Review by ofer-haim — APPROVED** [review]

File: filter-plugin/logstash-filter-aurora-mysql-guardium/README.md (hunk 2 above)
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-aurora-mysql-guardium/README.md:148]:
```suggestion
For the input configuration step, refer to the [CloudWatch_logs section](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md#configuring-a-CloudWatch-input-plug-in).
```

File: filter-plugin/logstash-filter-azure-apachesolr-guardium/README.md (hunk 1 above)
Comment (AlexaNahum) [reply filter-plugin/logstash-filter-azure-apachesolr-guardium/README.md:201]: please just double check in one instance that the links I'm adding work before applying to all.

File: filter-plugin/logstash-filter-documentdb-aws-guardium/README.md (hunk 4 above)
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-documentdb-aws-guardium/README.md:131]:
```suggestion
For the input configuration step, refer to the [CloudWatch_logs section](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md#configuring-a-CloudWatch-input-plug-in).
```

File: filter-plugin/logstash-filter-neo4j-guardium/README.md [hunk 8]
```diff
@@ -136,3 +136,6 @@ The Guardium universal connector is the Guardium entry point for native audit lo
 8. The "type" fields should match in the input and the filter configuration section. This field should be unique for  every individual connector added
 9. Click Save. Guardium validates the new connector, and enables the universal connector if it was disabled. After it is validated, it appears in the Configure Universal Connector page.
 
+## 5. Configuring the Neo4j filters in Guardium Insights
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
+In the ```Input configuration``` section, refer to the Filebeat section
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-neo4j-guardium/README.md:141]:
```suggestion
For the input configuration step, refer to the [Filebeat section](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md#Filebeat-input-plug-in-configuration).
```

File: filter-plugin/logstash-filter-onPremPostgres-guardium/README.md [hunk 9]
```diff
@@ -25,4 +25,8 @@ The plug-in is free and open-source (Apache 2.0). It can be used as a starting p
 ## Limitations
 	• Pgaudit logs Failed queries twice, one as success and one with actual Failure reason, so expect Failed queries to be seen in Full Sql Report
 	• The log entries which are generated even before the user is logged in or before the session starts, are currently not seen in SQL Error report
-	• When the database is created using UI option from pgadmin, currently sniffer fails to parse that audit log, so the entry cannot be seen in the Full Sql report
\ No newline at end of file
+	• When the database is created using UI option from pgadmin, currently sniffer fails to parse that audit log, so the entry cannot be seen in the Full Sql report
+
+## 5. Configuring the Postgres filters in Guardium Insights
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
+In the ```Input configuration``` section, refer to the Filebeat section
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-onPremPostgres-guardium/README.md:32]:
```suggestion
For the input configuration step, refer to the [Filebeat section](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md#Filebeat-input-plug-in-configuration).
```

File: filter-plugin/logstash-filter-postgres-guardium/AwsPostgres_README.md (hunk 6 above)
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-postgres-guardium/AwsPostgres_README.md:210]:
```suggestion
For the input configuration step, refer to the [CloudWatch_logs section](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md#configuring-a-CloudWatch-input-plug-in).
```

File: filter-plugin/logstash-filter-saphana-guardium/README.md [hunk 10]
```diff
@@ -293,3 +293,7 @@ In SAP HANA JDBC input plug-ins , we distribute load between two machines based
     ```
     select audit_log.event_status,audit_log.client_ip,audit_log.connection_id,audit_log.client_port,audit_log.timestamp,audit_log.event_action,audit_log.user_name,audit_log.port,audit_log.client_host,audit_log.service_name,audit_log.statement_string,audit_log.application_name,audit_log.host,audit_log.application_user_name,M_DATABASE.database_name from M_DATABASE,audit_log where M_DATABASE.HOST = audit_log.HOST and audit_policy_name not in ('MandatoryAuditPolicy') and mod(connection_id,2) = 1 and timestamp > :sql_last_value;
     ```
+
+## 5. Configuring the SAP HANA filters in Guardium Insights
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
+In the ```Input configuration``` section, refer to the Filebeat section
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-saphana-guardium/README.md:299]:
```suggestion
For the input configuration step, refer to the [Filebeat section](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md#Filebeat-input-plug-in-configuration).
```

File: filter-plugin/logstash-filter-yugabyte-guardium/README.md [hunk 11]
```diff
@@ -129,3 +129,6 @@ When UC will start collecting data, It may show two STAP statuses in the pattern
 8. The "type" fields should match in the input and the filter configuration section. This field should be unique for  every individual connector added.
 9. Click Save. Guardium validates the new connector, and enables the universal connector if it was disabled. After it is validated, it appears in the Configure Universal Connector page.
 
+## 5. Configuring the Yugabyte filters in Guardium Insights
+To configure this plug-in for Guardium Insights, follow [this guide.](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md)
+In the ```Input configuration``` section, refer to the Filebeat section
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-yugabyte-guardium/README.md:134]:
```suggestion
For the input configuration step, refer to the [Filebeat section](/docs/Guardium%20Insights/3.2.x/UC_Configuration_GI.md#Filebeat-input-plug-in-configuration).
```

**Review by AlexaNahum — APPROVED** [review]

File: filter-plugin/logstash-filter-aurora-mysql-guardium/README.md (hunk 2 above)
Comment (RefaelAdi) [reply filter-plugin/logstash-filter-aurora-mysql-guardium/README.md:148]: I'm not sure why, but I checked and it works and opened the valid URL.

Comment (RefaelAdi) [reply filter-plugin/logstash-filter-aurora-mysql-guardium/README.md:148]: fixed

**Pull request merged**